Curtains/Curtains.Application/CurtainsService/ImageToProductConversionService.cs

```diff
@@ -27,6 +31,10 @@ public IProduct Convert(ProductImageDTO productImage)
             };
         }
 
+        /// <summary>
+        /// Convert several images into IProducts
```

Comment:
What is IProducts. I can't understand.
Comment should contains max info and min incomprehensible words

Reply:
done